packages/node-resolve/src/index.js

```diff
@@ -222,7 +234,15 @@ export function nodeResolve(opts = {}) {
       importSpecifierList.push(importee);
       resolveOptions = Object.assign(resolveOptions, customResolveOptions);
 
-      let resolved = await resolveImportSpecifiers(importSpecifierList, resolveOptions);
+      const warn = (...args) => this.warn(...args);
+      const isRequire = opts && opts.custom && opts.custom['node-resolve'] && opts.custom['node-resolve'].isRequire;
```

Comment:
Should we warn when not using the RollupJS version that supports this?

Reply:
We actually need to check the version of the node-resolve plugin, which I think isn't even possible right now.

Reply:
Sorry I mean we need to check the version of the commonjs plugin.